content/json-ld-crud-api/latest/events/event-status.md

```diff
@@ -0,0 +1,116 @@
+---
+---
+
+# Event - status
+
+The status indicates whether an event takes place as planned, has been postponed or cancelled. The following values for 'status' are applicable:
+* `available`: the event takes place as planned. This is the default value when no status for the event or subEvent is included.
+* `temporarilyUnavailable`: the event has been postponed to a later date yet to be determined.
+* `unavailable`: the event has been cancelled.
+
+Optionally, a reason for the new status can be added. 
+
+When the status is set back to `available`, the reasons (if present) will be automatically removed.
+
+It is possible to set the status on top event level or on subEvent level:
+* **top event level**: for the whole event and all its subEvents (occurences). All subEvents will inherit the status from the top event level
+* **subEvent level**: for a one or multiple subEvent(s). The status on top event level will automtically be derived from the (different) statusses on subEvent level.
+
+## Set status on top event level
+
+It is possible to set the status on top event level for all calendarTypes: `single`, `multiple`, `permanent`, `periodic`. If the event has subEvents, all subEvents of the event will inherit the status (and reason) from the top event level.
+
+**Example**
+
+```
+"status":{
+   "type":"Unavailable",
+   "reason":{
+      "nl":"De lessenreeks is omwille van coronamaatregelen geannuleerd",
+      "fr":"Le série de cours a été annulé en raison de mesures corona"
+   }
+}
+```
+
+## Set status on subEvent level
+
+For calendarTypes `single` and `multiple` it is also possible to set the status on subEvent level.
+The status on top event level will be derived from the statusses on the subEvent level.
```

Comment:
```suggestion
The status on top event level will be derived from the statuses on the subEvent level.
```

Reply:
There are a few other occurrences of `statusses`, I think it has to be `statuses`.